### PR TITLE
Path space

### LINKED ISSUE #847: Spaces in paths aren't escaped properly on windows
*Automatically generated error report* 
UBports Installer Version: 0.2.3-beta 
Device: hammerhead
Channel: ubports-touch/16.04/devel
Package: exe
Operating System: Microsoft Windows [Vers�o 6.1.7601] ia32 
NodeJS version: v8.2.1 

Error log: http://paste.ubuntu.com/p/vg88NKpCcd/

### PR DESCRIPTION
**Pull request by NeoTheThird**
Fix #847 again, fml. Tested on windows, seems to work.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -263,7 +263,7 @@ let toolpath = global.packageInfo.package
 let processes = [];
 function execTool(tool, args, callback) {
   let pid = exec(
-    [path.join(toolpath, tool)].concat(args).join(" "),
+    [path.join(toolpath, tool).replace(" ", " ")].concat(args).join(" "),
```
Comment (Flohack74) [inline src/utils.js]: Erm does this replace a space with a space?

Comment (NeoTheThird) [reply src/utils.js]: lol, missed a backslash there. how the fuck did it work on my machine?  :D i created a new user with a space in the name and everything

Comment (Flohack74) [reply src/utils.js]: You going to push an update? :)

**Comment by NeoTheThird** [comment]
no longer needed

**Pull request closed without merging**